bin/plugin/commands/changelog.js

```diff
@@ -33,6 +33,47 @@ const manifest = require( '../../../package.json' );
  * @property {string}  milestone Milestone title.
  */
 
+/**
+ * Changelog normalization function, returning a string to use as title, or
+ * undefined if entry should be omitted.
+ *
+ * @typedef {(text:string,issue:IssuesListForRepoResponseItem)=>string|undefined} WPChangelogNormalization
+ */
+
+/**
+ * Mapping of label names to grouping heading text to be used in release notes,
+ * intended to be more readable in the context of release notes. Also used in
+ * merging multiple related groupings to a single heading.
+ *
+ * @type {Record<string,string>}
+ */
+const LABEL_GROUP_TITLES = {
+	Bug: 'Bug Fixes',
+	Regression: 'Bug Fixes',
+	Feature: 'Features',
+	Enhancement: 'Enhancements',
+	'New API': 'New APIs',
+};
```

Comment:
I think we should add "Experimental": "Experiments" here right?

Reply:
> I think we should add "Experimental": "Experiments" here right?

Yeah. Updated in 064a3dd.

Alternatively, the label "[Type] Experimental" could be renamed "[Type] Experiments". For future consideration.